### PR TITLE
Updated zh-cn doc

### PR DESCRIPTION
**Pull request by Rad586**

### CODE REVIEW

**Comment by melontini** [comment]
Thank you very much. There are some things that need fixing before I merge this.

File: docs/.vitepress/zh_cn.ts [hunk 1]
```diff
@@ -9,36 +9,34 @@ export const zh_cn = defineConfig({
...
+                text: 'Meta',
+                items: [
+                    { text: '设计', link: 'https://github.com/constellation-mc/commander/discussions/3' }
                 ]
             }
         ],
 
         search: {
             provider: 'local',
-            options: {
```
Comment (melontini) [inline docs/.vitepress/zh_cn.ts:22]: Vitepress doesn't provide built-in translations, so this is required.

File: docs/zh-cn/develop/Events.md [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+    .cancelTerm(Codec.INT)
+    .build(new Identifier("modid", "custom_event"));
+```
+
+通过使用 `extension()`，事件可以接受额外的参数。在指定扩展后，你可以返回自定义类型。
+
+```java
+EventType.builder()
+    .extension(Codec.STRING, subscriptions -> {
+        //handle data.
```
Comment (melontini) [inline docs/zh-cn/develop/Events.md:28]: Should be translated too.

File: docs/zh-cn/develop/Events.md [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+
+```java
+CustomEvent.EVENT.register((world, entity) -> runVoid(CUSTOM_EVENT, world, () -> makeContext(world, entity, entity.getPos())));
+```
+
+```java
+private static LootContext makeContext(ServerWorld world, Entity entity, Vec3d origin) {
+    LootContextParameterSet.Builder builder = new LootContextParameterSet.Builder(world);
+    builder.add(THIS_ENTITY, entity).add(ORIGIN, origin);
+    return new LootContext.Builder(builder.build(LootContextTypes.COMMAND)).build(null /*Optional.empty() in 1.20.4*/);
```
Comment (melontini) [inline docs/zh-cn/develop/Events.md:50]: Should be translated too.

File: docs/zh-cn/develop/Events.md [hunk 4]
```diff
@@ -0,0 +1,69 @@
...
+}
+```
+
+如果你指定了扩展，或者使用了不受支持的返回类型，就需要编写自定义的解析逻辑。
+
+为了编写这一逻辑，你需要创建 `EventContext`。`EventContext` 用于把执行参数传递给命令。
+
+```java
+EventContext context = EventContext.builder(type)
+        .addParameter(EventKey.LOOT_CONTEXT, /*instance of loot context*/)
```
Comment (melontini) [inline docs/zh-cn/develop/Events.md:60]: Should be translated too.

**Review by melontini — CHANGES_REQUESTED** [review]

File: docs/zh-cn/develop/Commands.md [hunk 5]
```diff
@@ -0,0 +1,58 @@
...
+    }
+
+    @Override
+    public CommandType type() {
+        return null;
+    }
+}
+```
+
+接下来，我们再为这个命令[编码器](https://forge.gemwire.uk/wiki/Codecs)创建一个编码器。
```
Comment (melontini) [inline docs/zh-cn/develop/Commands.md:27]: I wonder if there's a translated resource about Codecs.

Comment (Rad586) [reply docs/zh-cn/develop/Commands.md:27]: Unfortunately, I didn't find any translated resource about this.

File: docs/.vitepress/zh_cn.ts (hunk 1 above)
Comment (Rad586) [reply docs/.vitepress/zh_cn.ts]: Added back

**Review by melontini — APPROVED** [review]

**Pull request merged**